lib/net/common.js

```diff
@@ -91,6 +91,26 @@ exports.MAX_MESSAGE = 8 * 1000 * 1000;
 
 exports.BAN_TIME = 24 * 60 * 60;
 
+/**
+ * Amount of time after which to reset DOS peers.
+ * @const {Number}
+ * @default
+ */
+
+exports.DOS_RESET_TIME = 30;
+
+/*
+ * Precomputed decay factor for first 1024 seconds.
+ * @const {Object}
+ * @default
+ */
+
+const dosDecayFactor = {};
+for (let i=0; i < 1024; i++)
+    dosDecayFactor[i] = Math.exp(-1.0 * i * Math.log(2));
```

Comment:
nit: what do you think about a comment referencing the origin of this exponential function?

Reply:
@tuxcanfly did you see this comment?

Reply:
Sorry, added a note and the formula for explanation.

Reply:
@tuxcanfly given the updated calculation with a dynamically calculated half-life, should we just remove this pre-computation table? It is not being used anymore.

Reply:
Yeah it's not being used atm. But it would nevertheless be useful if it could use a given half-life, maybe after config is parsed for example, as a minor optimization. Will benchmark and move or remove it depending on the results.